core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -336,6 +334,9 @@ public void provideOutcome(AppCreator ctx) throws AppCreatorException {
                     throw new AppCreatorException("GRAALVM_HOME was not set");
                 }
             }
+            if (isThisGraalVMRCObsolete(graalvmHome)) {
+                throw new AppCreatorException("you must update GraalVM version");
```

Comment:
The `isThisGraalVMRCObsolete` is logging a visible error but it's not failing. I decided to not fail hard because the check would need to be 100% reliable - while sometimes one wants to overrule it and test on a different version, or possibly a newer version than what we hardcoded.

I think we could explore the idea of throwing an hard error, but only for very old versions which are well known to not be compatible.

Reply:
I would throw a hard error for the versions older than the one we officially support and just a warning for versions newer than the one we tested.

Reply:
Thank for reviews. why I added throwing a hard error is that I got error when I tried to run quarkus on GraalVM 1.0.0-rc16.
```
[INFO] [io.quarkus.creator.phase.nativeimage.NativeImagePhase] /Library/Java/JavaVirtualMachines/graalvm-ce-1.0.0-rc16/Contents/Home/bin/native-image -J-Djava.util.logging.manager=org.jboss.logmanager.LogManager -J-Dio.netty.leakDetection.level=DISABLED -J-Dvertx.disableDnsResolver=true -J-Dio.netty.noUnsafe=true --initialize-at-build-time= -H:InitialCollectionPolicy=com.oracle.svm.core.genscavenge.CollectionPolicy$BySpaceAndTime -jar getting-started-1.0-SNAPSHOT-runner.jar -J-Djava.util.concurrent.ForkJoinPool.common.parallelism=1 -H:FallbackThreshold=0 -H:+ReportExceptionStackTraces -H:+PrintAnalysisCallTree -H:-AddAllCharsets -H:EnableURLProtocols=http,https --enable-all-security-services -H:-SpawnIsolates -H:+JNI --no-server -H:-UseServiceLoaderFeature -H:+StackTrace
Error: Unrecognized option: --initialize-at-build-time=
```

It is occured because old version GraalVM didn't support ```--initialize-at-build-time=``` option.
I thought it would be more useful if it show the reason and stop following processing.
For now, I correct to show warning if GraalVM is newer than current suppored version.
please check this out and any comment.